docs/userGuide/plugins/web3Form.md

```diff
@@ -0,0 +1,89 @@
+### Plugin: Web3Forms 
+
+This plugin allows you to create forms whose response will be sent directly to your email, using the [Web3Form](https://web3forms.com/) API. 
+
+To set it up, get an access key from [Web3Forms](https://web3forms.com/). Then add`web3Form` to your site's plugin, and add the `accessKey` parameter via the `pluginsContext`.
+
+| Name      | Type     | Default                             | Description |
+|-----------|----------|-------------------------------------|-------------|
+| accessKey | `String` ||  accessKey is provided by Web3Form. |
+
+```js {heading="site.json"}
+{
+  ...
+  "plugins": [
+    "web3Form"
+  ],
+  "pluginsContext": {
+    "web3Form": {
+      "accesskey": "YOUR_WEB_3_FORM_ACCESS_KEY", // replace with your Web3Form access key 
```

Comment:
Also is this code block language is labelled `js` instead of `json` to support comments? 🤔

Reply:
Opps sorry I copied it from documentation on other plugins which was using js.

Reply:
I believe so. Changing it to JSON will get warnings regarding comments and regarding the trailing commas in the start

Reply:
Ah I see, in that case I think it's fine to keep it as `js` to maintain consistency with the other plugins especially since the difference in syntax highlighting would be visible to users